pkgs/desktops/xfce/art/xfwm4-themes.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  p_name  = "xfwm4-themes";
+  ver_maj = "4.10";
+  ver_min = "0";
+
+  src = fetchurl {
+    url = "mirror://xfce/src/art/${p_name}/${ver_maj}/${name}.tar.bz2";
+    sha256 = "0xfmdykav4rf6gdxbd6fhmrfrvbdc1yjihz7r7lba0wp1vqda51j";
+  };
+  name = "${p_name}-${ver_maj}.${ver_min}";
+
+  meta = {
```

Comment:
I meant if you feel it is important, one could write a script which downloads ```src``` of every derivation in nixpkgs and check ```meta.license``` against the strings in the text files in the source tarball and then make PRs fixing licenses.
I'd expect there are too many files to fix so it should be a bot job.